Rakefile

```diff
@@ -1,5 +1,5 @@
 require "bundler/gem_tasks"
-require 'metric_fu'
+require 'metric_fu' if RUBY_VERSION >= '2.1.0'
```

Comment:
I see you have the version condition in the `Gemfile`, but should this be more of a `defined?` condition here?